ios/RNSentry.m

```diff
@@ -140,19 +140,23 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
     NSLog(@"Bridge call to: deviceContexts");
+    NSMutableDictionary<NSString *, id> *contexts = [NSMutableDictionary new];
```

Comment:
`l`: If you do it like this and the comment below, you would avoid allocating an unneeded NSMutableDictionary .
```suggestion
    NSDictionary<NSString *, id> *contexts = nil;
```